src/Workspaces/Core/Portable/Extensions/OperationExtensions.cs

```diff
@@ -144,6 +146,20 @@ public static ValueUsageInfo GetValueUsageInfo(this IOperation operation)
             {
                 return ValueUsageInfo.Write;
             }
+            else if (operation.Parent is IVariableInitializerOperation variableInitializerOperation)
+            {
+                if (variableInitializerOperation.Parent is IVariableDeclaratorOperation variableDeclaratorOperation)
+                {
+                    switch (variableDeclaratorOperation.Symbol.RefKind)
+                    {
+                        case RefKind.Ref:
+                            return ValueUsageInfo.ReadableWritableReference;
+
+                        case RefKind.RefReadOnly:
+                            return ValueUsageInfo.ReadableReference;
+                    }
```

Comment:
@mavasani Should we include the `default` case here rather than rely on fall-through?

Reply:
Would the default be something other then Read?